src/main/app/src/app/zaken/zaken-werkvoorraad/zaken-werkvoorraad.component.ts

```diff
@@ -284,7 +284,7 @@ export class ZakenWerkvoorraadComponent
       }));
       const subscription = this.websocketService.addListener(
         Opcode.UPDATED,
-        ObjectType.ZAAK,
+        ObjectType.ZAAK_ROLLEN,
```

Comment:
Just checking: ZAAK_ROLLEN is what you want to be checking for here? I can't make that up from the function name `handleWorkflow` which seems to me to be too generic? Why only add a listener for ZAAK_ROLLEN events here? I must be missing something. ;-)

Reply:
I'll think about a better function name. It's listening to ZAAK_ROLLEN, because that's what get's updated by the assignment/releasement process. If we receive that, it means that a OpenNotificatie was received an the Solr index has been updated (but possibly not committed yet)

Reply:
ah yes. But there are many more types of changes that can be received from OpenNotificaties and for which also the Solr index has been updated. That's more my point. E.g. when a new zaak is created etc etc. But indeed, this one is the one you will get for assigning and releasing zaken.

Reply:
Also note by the way that such a ZAAK_ROLLEN notification _can_ be caused by assigning or releasing zaken from the list (=werkvoorraad) but of course this does not have to be. There can be many other causes for such a notification type. E.g. people could simple add new new zaak role directly in OpenZaak. Or also: when you assign a zaak to yourself or someone else, not from the werkvoorraad. etc etc. Is that a possible issue here or not?

Reply:
Not really, I think. Indeed there could be a timing issue but I don't think it would matter much. Potentially, another notification for the same zaak could cause the progress indicator to increase, but in the end the changes are commited to solr and the werkvoorraad is refereshed, so all changes made are taken into account. The more cases you assign/release, the more chance that there is an unrelated notification, but the less chance that it will matter in the end.